source/coll_jaw.f90

```diff
@@ -0,0 +1,225 @@
+! ================================================================================================ !
+!  Collimation Jaw Fit Module
+! ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!  V.K. Berglyd Olsen, BE-ABP-HSS
+!  Created: 2019-08-01
+!  Updated: 2019-08-02
+! ================================================================================================ !
+module coll_jaw
+
+  use floatPrecision
+  use numerical_constants, only : zero, one
+
+  implicit none
+
+  type, private :: jaw_fitType
```

Comment:
It's called "Type" because it's the type declaration. I call the storage "List" or "Store". It's just the naming convention I've been using in various modules.

Reply:
then, if we were in C, you would have called it `jaw_fitStruct`.
IMHO, I don't think it is a smart naming convention

Reply:
I disagree, I find that very descriptive, but ... * shrug *

Reply:
sorry, I am simply saying that this `type` contains the fitting parameters, and `jaw_fitType` does not convey such an idea; hence, the request of changing the name. We can have `jaw_fitSpecs_type`, if you prefer, it is fine with me.